starter-workflows/code-scanning/scorecards.yml

```diff
@@ -0,0 +1,69 @@
+name: Scorecards supply-chain security
+on: 
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    # Weekly on Saturdays.
+    - cron: '30 1 * * 6'
+  push:
+    branches: [ $default-branch ]
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  analysis:
+    name: Scorecards analysis
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+    
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v1
+
+      - name: "Run analysis"
+        # TODO: update with a hash once we have a release.
+        uses: ossf/scorecard-action@feat/rempol
```

Comment:
cc @naveensrinivasan we're going to need to use the more official image from docker registry

Reply:
Why docker registry? Is there a requirement?

Reply:
my bad, I mean GitHub registry!

Reply:
created this a few days back too keep track of it https://github.com/ossf/scorecard-action/issues/12. It should not change much from what you already did. Let me know on the issue if there's something else I'm missing.